boards/nxp/frdm_mcxw71/doc/index.rst

```diff
@@ -194,6 +213,19 @@ Please consider changing ``COMxx`` on Windows or ``ttyxx`` on Linux to the seria
 
 The NBU file can be found in : ``<zephyr workspace>/modules/hal/nxp/zephyr/blobs/mcxw71/mcxw71_nbu_ble.sb3``
 
+For dynamic (bluetooth + ieee802154) NBU:
+.. code-block:: console
+
+   # On Windows
+   blhost.exe -p COMxx -- receive-sb-file mcxw71_nbu_ble_15_4_dyn.sb3
+
+   # On Linux
+   ./blhost -p /dev/ttyxx -- receive-sb-file mcxw71_nbu_ble_15_4_dyn.sb3
+
+Please consider changing ``COMxx`` on Windows or ``ttyxx`` on Linux to the serial port used by your board.
+
+The NBU file can be found in : ``<zephyr workspace>/modules/hal/nxp/zephyr/blobs/mcxw71/mcxw71_nbu_ble_15_4_dyn.sb3``
```

Comment:
There is a helper for doing this using tabs, which renders to a slightly cleaner interface.

https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/modules/thrift/hello/README.rst?plain=1#L65

Reply:
Done, thank you